admin/commands/storage/read_range_blocks.go

```diff
@@ -27,6 +27,11 @@ func (c *ReadRangeBlocksCommand) Handler(ctx context.Context, req *admin.Command
 		return nil, err
 	}
 
+	limit := uint64(10001)
```

Comment:
`10001` instead of `10000`, because `10000` won't allow a range from `10000, 20000`, which is easier to type than `10001, 20000`

Reply:
can you add that as a comment. it won't be obvious to the next person that reads the code

Reply:
+1.  also, maybe convert it into an const.  same elsewhere